tests/test_publish.py

```diff
@@ -95,34 +113,158 @@ def fixture_publish_db():
         [0, 1, 1, 2],
         index=audformat.filewise_index(db.files[:4]),
     )
+    audeer.mkdir(audeer.path(db_root, 'extra/folder'))
+    audeer.touch(audeer.path(db_root, 'extra/file.txt'))
+    audeer.touch(audeer.path(db_root, 'extra/folder/file1.txt'))
+    audeer.touch(audeer.path(db_root, 'extra/folder/file2.txt'))
+    db.attachments['file'] = audformat.Attachment('extra/file.txt')
+    db.attachments['folder'] = audformat.Attachment('extra/folder')
     db.save(
-        DB_ROOT_VERSION['1.0.0'],
+        db_root,
         storage_format=audformat.define.TableStorageFormat.PICKLE,
     )
     audformat.testing.create_audio_files(db)
 
-    # Extend version 2.0.0 by a new file with a path >260 characters
+    # Version 2.0.0
+    #
+    # Changes:
+    #   * Added: new file with a path >260 characters,
+    #   * Removed: 1 attachment file
```

Comment:
Maybe in addition also add a new attachment file as I don't see this case is covered in any of the versions yet.

Reply:
I added now `extra/folder/file3.txt` in this version. It is also of interest as the file has the same name as the existing attachment file in the sub-folder: `extra/folder/sub-folder/file3.txt`